index.html

```diff
@@ -161,6 +161,36 @@ <h2></h2>
                             <td><img id="Sushie" class="unselected partner" src="images/partners/sushie.png"></td>
                             <td><img id="Lakilester" class="unselected partner" src="images/partners/lakilester.png"></td>
                         </tr>
+                        <tr id="DungeonEntranceRow" style="display:none">
```

Comment:
Duplicate IDs is bad practice, there shouldn't be a reason in this case to have DungeonEntranceRow as an ID instead of a class. That said, this works.